packaging/debian/control

```diff
@@ -0,0 +1,20 @@
+Source: ricochet
+Maintainer: Peter Ludikovsky <peter@ludikovsky.name>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>=9), libssl-dev (>=1.0.0), pkg-config, gcc (>=4.9) | clang (>=3.5), make (>=4.0), libprotobuf-dev, protobuf-compiler, qt5-qmake (>=5.1.0), qtbase5-dev (>=5.1.0), qtchooser (>=5.1.0), qtbase5-dev (>=5.1.0), qttools5-dev-tools (>=5.1.0), qtdeclarative5-dev (>=5.1.0), qtmultimedia5-dev (>=5.1.0), qml-module-qtquick-controls (>=5.1.0), qml-module-qtquick-dialogs (>=5.1.0), tor (>=0.2.5.12)
```

Comment:
Is it actually necessary to build-depend on `gcc | clang`, `make`, etc?

Also, these:

```
qml-module-qtquick-controls (>=5.1.0), qml-module-qtquick-dialogs (>=5.1.0), tor (>=0.2.5.12)
```

Are not build-dependencies; they should be install dependencies instead.